core/prob_dist.py

```diff
@@ -32,6 +32,13 @@ def alphabet(self):
     def prob_list(self):
         return [self.prob_dict[s] for s in self.alphabet]
 
+    @classmethod
+    def get_sorted_prob_dist(cls, prob_dict, reverse=True):
+        """
+        Returns ProbabilityDist class object with sorted probabilities
```

Comment:
it would be good to clarify the default sort order (i.e. increasing vs decreasing... the reverse flag is not very intuitive